package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@processout/picasso",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
You should use `npm version` to update the package version number. This ensures the `package-lock.json` file is properly updated. For more info, refer to the [npm docs](https://docs.npmjs.com/updating-your-published-package-version-number) 🙂